include/matx/transforms/cub.h

```diff
@@ -563,7 +563,9 @@ inline void ExecSort(OutputTensor &a_out,
 
 #ifdef __CUDACC__
   static_assert(is_tensor_view_v<InputOperator>, "Sorting only accepts tensors for now (no operators)");
-  MATX_ASSERT_STR(a.IsContiguous(), matxInvalidType, "Tensor must be contiguous in memory for sorting");
+  if (! a.IsContiguous()) {
```

Comment:
We eventually should have checks that are for both debug and release

Reply:
Agreed. In this case, I expect that we would currently never throw this exception because the caller of this function will allocate a contiguous tensor if necessary. I left it in place in case we rework things in the future to avoid the temporary allocation in some cases (e.g., by using CUB's merge sort).